bake/bake.go

```diff
@@ -534,7 +535,8 @@ func (c Config) newOverrides(v []string) (map[string]map[string]Override, error)
 		}
 
 		pattern := keys[0]
-		if len(parts) != 2 && keys[1] != "args" {
+		attrName := strings.TrimSuffix(keys[1], "+")
+		if len(parts) != 2 && attrName != "args" {
```

Comment:
out of scope: why isn't the `labels` behaving the same

Reply:
I guess `annotations` and `labels` don't behave like `args` because key can contain invalid chars when used with override like `.` Reminds me of this conversation with @jsternberg https://github.com/docker/buildx/pull/2997#pullrequestreview-2618430667

Reply:
Why would invalid characters matter, the "parser" uses `splitn(,3)`. Annotations would be different as that is an array, not a map.